### PR TITLE
added BT & RT env for newrelic-java-agent

### PR DESCRIPTION
**Pull request by NeerajK007**
added BT & RT env for newrelic-java-agent

### CODE REVIEW

File: newrelic/0.0.1/besman-newrelic-java-agent-BT-env-config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+BESMAN_ORG: Be-Secure
+BESMAN_OSSP: newrelic-java-agent
+BESMAN_OSSP_CLONE_PATH: $HOME/$BESMAN_OSSP
+BESMAN_ANSIBLE_ROLES_PATH: $BESMAN_DIR/tmp/$BESMAN_OSSP/roles
+BESMAN_ANSIBLE_ROLES: Be-Secure/ansible-role-bes-java
```
Comment (asa1997) [inline newrelic/0.0.1/besman-newrelic-java-agent-BT-env-config.yaml:6]: We need a package manager as well as an ide.

File: newrelic/0.0.1/besman-newrelic-java-agent-RT-env-config.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+BESMAN_ORG: Be-Secure
+BESMAN_OSSP: newrelic-java-agent
+BESMAN_OSSP_CLONE_PATH: $HOME/$BESMAN_OSSP
+BESMAN_ANSIBLE_ROLES_PATH: $BESMAN_DIR/tmp/$BESMAN_OSSP/roles
+BESMAN_ANSIBLE_ROLES: Be-Secure/ansible-role-bes-java
```
Comment (asa1997) [inline newrelic/0.0.1/besman-newrelic-java-agent-RT-env-config.yaml:6]: Add a package manager and an ide.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@NeerajK007 Please look at the comments.

**Comment by NeerajK007** [comment]
Closing this as suggested feedback is incorporated in another pull request

**Pull request closed without merging**